purchase_report_custom_hls/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Purchase Order Report Custom",
+    "version": "12.0.1.0.0",
+    "category": "Purchase",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "license": "LGPL-3",
+    "installable": True,
+    "depends": ["purchase", "web"],
```

Comment:
```suggestion
    "depends": ["purchase"],
```
`web` is a distant dependency of `purchase`, therefore no need to be included.

Reply:
Thank you, I confirmed it works without web depends.

Reply:
I edited depends, add report_common_hls.